hypex/dataset/backends/abstract.py

```diff
@@ -342,6 +345,9 @@ def merge(
     def drop(self, labels: FieldKeyTypes = "", axis: int = 1) -> Any:
```

Comment:
`Optional[FieldKeyTypes]` and "" to None

Reply:
fixed